src/main/java/pl/sda/javapoz/controller/LoginController.java

```diff
@@ -3,17 +3,12 @@
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.GetMapping;
 
-/**
- * Created by pablo on 24.03.17.
- */
+
 @Controller
 public class LoginController {
 
     @GetMapping("login")
```

Comment:
Tutaj sie zastanawiam czy czasem na /login nie powinien być GetMapping.

Reply:
w sensie  że brakuje tego Schragstrich przed login czyli /login

Reply:
aha. Poprawilem to przed chwila